CHANGELOG.md

```diff
@@ -51,6 +52,25 @@ All notable changes to "Azure Toolkit for IntelliJ IDEA" will be documented in t
   - [3.0.7](#307)
   - [3.0.6](#306)
 
+## 3.39.0
+
+### Added
+- Support Azure Functions with Java 11 runtime(Preview)
+- Support authentication with Azure CLI credentials
+
+### Changed
+- Show Apache Spark on Cosmos node by default no matter if there is SoC clusters under user's subscription
```

Comment:
My bad. Please replace `no matter if there is SoC clusters under user's subscription ` with `no matter whether there are SoC clusters under user's subscription or not`